Arm/Insts/DPI/PC_rel_addressing.lean

```diff
@@ -20,7 +20,7 @@ def exec_pc_rel_addressing (inst : PC_rel_addressing_cls) (s : ArmState) : ArmSt
   let imm := if inst.op = 0#1 then
                 signExtend 64 (inst.immhi ++ inst.immlo) -- ADR
              else
-                signExtend 64 ((inst.immhi ++ inst.immlo) <<< 12) -- ADRP
+                signExtend 64 (inst.immhi ++ inst.immlo ++ (BitVec.zero 12)) -- ADRP
```

Comment:
Ugh, sorry about this bug and thanks for fixing it! I sometimes forget that bitvec operations are fixed-width. :)